DYDX-Bots/Roles_Monitor/README.md

```diff
@@ -0,0 +1,53 @@
+# Role Changes Monitor Bot
+
+## Description
+
+This bot detects changes in roles in both the Safety Module and Liquidity Module contracts.
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- DYDX-18-1
+  - Fired when a `RoleAdminChanged` event has been fired from either the Safety or Liquidity Module.
+  - Severity is always set to "Info".
+  - Type is always set to "Info".
+  - Metadata includes:
+    - `role`: role that has changed.
+    - `previousAdminRole`: admin role for `role` before being replaced.
+    - `newAdminRole`: new admin role for `role` that replaced `previousadminRole`.
+
+- DYDX-18-2
+  - Fired when a `RoleGranted` event has been fired from either the Safety or Liquidity Module.
+  - Severity is always set to "Info".
+  - Type is always set to "Info".
+  - Metadata includes:
+    - `role`: role that has been granted.
+    - `account`: the account that has been granted the role.
+    - `sender`: the account that originated the contract call.
+
+- DYDX-18-3
+  - Fired when a `RoleRevoked` event has been fired from either the Safety or Liquidity Module.
+  - Severity is always set to "Info".
+  - Type is always set to "Info".
+  - Metadata includes:
+    - `role`: role that has been revoked..
+    - `account`: the account that has been granted the role.
+    - `sender`: the account that originated the contract call.
+
+## Test Data
+The bot behavior can be verified with the following contracts on the Kovan ETH testnet:
+
```

Comment:
I suggest adding some mainnet test transactions since those are available, here's the ones i got while checking: 

- 0xd0609f2a45bc00da0de1e5f47c2e7a625ad25ca50011ecf143706cfc16e0b5ee - `RoleGranted` event on Liquidity Module.

- 0xa07da3a1bb83f82086c87348955904d01dbe98f99ef32823be6d711675a0d5c7 - `RoleRevoked` event on Liquidity Module.

- 0x208a8221ff67f3c8c0d5e108c1e3291bce012462ba879d430bf355ad583d2c1f - multiple `RoleAdminChanged`, `RoleGranted` events on Safety Module.